funsite/utils.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from collections import namedtuple
+
+from django.utils.translation import pgettext, ugettext as _
+
+BreadcrumbsItem = namedtuple('BreadcrumbsItem', ['path', 'name'])
+
+
+def breadcrumbs(url, current_page):
+    """
+    Return a list of pair (path, i18ned name) of the breadcrumb for current url
+    The tricky part is the replacement of edX's courses page by our.
+    See tests.test_breadcrumbs to understand awaited behaviour
```

Comment:
Il vaut mieux documenter ici le comportement attendu plutôt que de pointer vers le test. Par exemple:

    Returns:
        result (BreadcrumbsItem list): list of maximulm length 3. The first item is the home page. The last item is the current page. The middle item, if present, points to the parent page of the current page.